### PR TITLE
test

### PR DESCRIPTION
**Pull request by OleksandrFEdev**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,18 @@
...
 function calculateRentalCost(days) {
-  // write code here
+  const carRent = 40;
+  let fullCost = days * carRent;
+  const moreThan7DaySale = 50;
+  const moreThan3DaySale = 20;
+
+  if (days >= 7) {
+    fullCost -= moreThan7DaySale;
+  } else if (days >= 3) {
```
Comment (VitaliyBodnar) [inline src/calculateRentalCost.js:38]: 7 and 3 are a magic numbers, so make sense to move them to the constants

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carRent = 40;
```
Comment (VitaliyBodnar) [inline src/calculateRentalCost.js:31]: it's better to use UPPER_CASE for static variables

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

File: src/calculateRentalCost.js (hunk 2 above)
Comment (lerastarynets) [reply src/calculateRentalCost.js:31]: not fixed

**Review by lerastarynets — APPROVED** [review]
Great job overall! Left one tiny suggestion for the improvement

**Pull request closed without merging**